examples/hnas/example_mlp.py

```diff
@@ -0,0 +1,228 @@
+import autogluon.core as ag
+import time
+import logging
+import yaml
+import argparse
+
+import torch
+import torch.nn as nn
+
+from torch.utils.data.sampler import SubsetRandomSampler
+
+from torchvision import datasets
+from torchvision import transforms
+
+
+@ag.args(
+    n_units_1=ag.space.Int(lower=4, upper=1024),
+    n_units_2=ag.space.Int(lower=4, upper=1024),
+    dropout_1=ag.space.Real(lower=0, upper=.99),
+    dropout_2=ag.space.Real(lower=0, upper=.99),
+    learning_rate=ag.space.Real(lower=1e-6, upper=1, log=True),
+    batch_size=ag.space.Int(lower=8, upper=128),
+    wd=ag.space.Real(lower=1e-8, upper=1, log=True),
+    epochs=27
+)
+def objective_function(args, reporter, **kwargs):
```

Comment:
You can drop the **kwargs. I changed the promotion-based HB not to supply resume_from this way anymore.

Reply:
ok dropped